example-get-started-experiments/code/.github/workflows/deploy-model-template.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Model (Template)
+
+on:
+  # the workflow is triggered whenever a tag is pushed to the repository
+  push:
+    tags:
+      - "*"
+  # we can also run the workflow manually...
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Manual version name'
+        required: true
+        type: string
```

Comment:
@dberenbaum I think we can remove it completely from both workflows. I think that @daavoo intended to have a manual trigger but it was probably never fully implemented...I just copied it without really studying it too much.

Reply:
I don't mind the manual trigger. I'm just confused by the `version` input.

Reply:
The manual trigger does not really do anything anyway since GTO won't parse any tags and so the second job will always be skipped... I think it makes more sense to just remove it entirely.